tvbextxircuits/nb_generator.py

```diff
@@ -5,16 +5,23 @@
 # (c) 2022-2023, TVB Widgets Team
 #
 
-import importlib
 import json
 import os
+import sys
 import nbformat
+import importlib
 from tvb.simulator.integrators import HeunDeterministic
 from tvb.simulator.models.oscillator import Generic2dOscillator
-
 from xai_components.base_tvb import ComponentWithWidget
 
+from xai_components.logger.builder import get_logger
+
+LOGGER = get_logger(__name__)
+USE_ABSOLUTE_PATHS = sys.platform.startswith('win')
 NOTEBOOKS_DIR = 'TVB_generated_notebooks'
+if USE_ABSOLUTE_PATHS:
+    NOTEBOOKS_DIR = os.path.abspath('TVB_generated_notebooks')
+
```

Comment:
The absolute paths were not working to open the widget PhasePlane notebook on Unix.
Thus, we maintain the previous behaviour with relative paths for Unix (this included ebrains lab) and we have the absolute paths for Windows.